cmd/prom2json/main.go

```diff
@@ -19,10 +19,9 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/prometheus/log"
-
 	dto "github.com/prometheus/client_model/go"
 
+	"github.com/prometheus/common/log"
```

Comment:
This line should be where it was before the change (we sort in blocks: stdlib, external, named external, internal, named internal).